library/src/main/java/com/novoda/downloadmanager/LiteDownloadService.java

```diff
@@ -17,19 +16,26 @@
 
     private static final long TEN_MINUTES_IN_MILLIS = TimeUnit.MINUTES.toMillis(10);
     private static final String WAKELOCK_TAG = "WakelockTag";
-    private static final String NOTIFICATION_TAG = "download-manager";
 
     private ExecutorService executor;
     private IBinder binder;
     private PowerManager.WakeLock wakeLock;
-    private NotificationManagerCompat notificationManagerCompat;
 
     @Override
     public void onCreate() {
         super.onCreate();
         executor = Executors.newSingleThreadExecutor();
         binder = new DownloadServiceBinder();
-        notificationManagerCompat = NotificationManagerCompat.from(this);
+    }
+
+    @Override
+    public void start(int id, Notification notification) {
+        startForeground(id, notification);
```

Comment:
This is the only weird thing now I guess @danybony but probably a fine tradeoff

Reply:
yeah, I think it's acceptable 👍